connector/oidc/oidc.go

```diff
@@ -211,6 +229,15 @@ func (c *oidcConnector) LoginURL(s connector.Scopes, callbackURL, state string)
 	if s.OfflineAccess {
 		opts = append(opts, oauth2.AccessTypeOffline, oauth2.SetAuthURLParam("prompt", c.promptType))
 	}
+
+	if len(c.additionalAuthRequestParams) > 0 {
+		for k, v := range c.additionalAuthRequestParams {
+			if contains(managedAuthParams, k) {
```

Comment:
`golang.org/x/exp` is already a dependency of this project. You could use [golang.org/x/exp/slices.Contains](https://pkg.go.dev/golang.org/x/exp/slices#Contains) instead.

Reply:
Also, I wonder if this check would be better of in the connector open function as a validation.